node/src/command.rs

```diff
@@ -571,6 +568,23 @@ pub fn run() -> Result<()> {
 					if config.role.is_authority() { "yes" } else { "no" }
 				);
 
+				if chain_spec.is_dev() {
+					return if chain_spec.is_crab() {
```

Comment:
Curious about the relationship between the `start_dev_node` and the `start_parachain_node` next, are they doing duplicated things?  Seems they all `start_network` at the end.

Reply:
They do part of the same thing, but `start_dev_node` can only be used for devnet, because it will be finalized instantly